### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by HUNTERVEDANSH**
This python code helps to extract data from Hokuyo UTM-30LX LRF. It also helps to find different dimensions of the object in front of it.

### Describe your change:
I want to add a file that can work as an algorithm to solve data extraction problem in Hokuyo LRFs. 

* [x] Add an algorithm?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by HUNTERVEDANSH** [comment]
How to get my file reviewed ?

File: hukoyu_laser_data_extraction.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python
+import rospy # Python library for ROS
```
Comment (poyea) [inline hukoyu_laser_data_extraction.py:2]: This seems like a robotic program. Could you put it in an appropriate folder?

Comment (HUNTERVEDANSH) [reply hukoyu_laser_data_extraction.py:2]: Thank you. Can you pls help with that?

**Comment by cclauss** [comment]
In preparation for that denial-of-service attack that is [Hacktoberfest](https://hacktoberfest.com), I am closing all pull requests that have failing tests.

**Pull request closed without merging**